.circleci/gradle.properties

```diff
@@ -1,4 +1,4 @@
-org.gradle.jvmargs=-Xmx4g -Dkotlin.compiler.execution.strategy="in-process"
+org.gradle.jvmargs=-Xmx5g -Dkotlin.compiler.execution.strategy="in-process"
```

Comment:
I noticed that our `build_instrumented` job seemed to be having a hard time completing, so I'm bumping up the memory to help it out.